frontend/test/metabase/scenarios/admin/permissions/sandboxes.cy.spec.js

```diff
@@ -596,18 +598,11 @@ describeWithToken("formatting > sandboxes", () => {
           /**
            * Helper function related to this test only!
            */
-          function runAndSaveQuestion({ question, sandboxValue } = {}) {
-            // Run the question
+          function visitQuestion({ question, sandboxValue } = {}) {
+            // Visit & run the question
             cy.visit(`/question/${question}?sandbox=${sandboxValue}`);
             // Wait for results
             cy.wait("@cardQuery");
-            // Save the question
-            cy.findByText("Save").click();
-            modal().within(() => {
-              cy.button("Save").click();
-            });
-            // Wait for an update so the other queries don't accidentally cancel it
-            cy.wait("@questionUpdate");
```

Comment:
@daltojohnso wasn't the whole purpose of the workaround to:
1. run the question (obtain the metadata)
2. then save it

It's been a while since I was working on this, but it seems that this diff defeats the original logic. What am I missing?